canonical-websites/www.ubuntu.com
# Add a french takeover for vmware to openstack

--- LIVE FOR THE 20 MARCH ---

## Done

- Added a takeover for the french version of the VMware to OpenStack

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Change your browser to use french
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare the text to the [brief](https://docs.google.com/document/d/14Lzlydn9YNjTm-x-ACuKSE5xJaBe3xyHzFakQ1oRIVY/edit)


## Issue / Card

Fixes #4839

## Screenshots

![image](https://user-images.githubusercontent.com/441217/54479844-56d9b000-4819-11e9-8aab-f25992186762.png)
